internal/executor/testdata/persistent-worker-graceful-termination/.cirrus.yml

```diff
@@ -0,0 +1,4 @@
+persistent_worker: {}
+
+unix_task:
+  script: sleep 86400
```

Comment:
What do you think of a bit more complex example? For example:

```yaml
persistent_worker: {}

unix_task:
  deattach_script: sleep 86400 &
  ps_script: ps
  sleep_script: sleep 86400
```

So in the test we can parse `pid` from `deattach` script and verify it's not longer running after the agent is killed?

Reply:
I think we only need to check that the agent terminated gracefully here.

What you've demonstrated above belongs to the agent's tests.